memote/suite/tests/test_basic.py

```diff
@@ -375,34 +375,6 @@ def test_find_constrained_transport_reactions(read_only_model):
                                                truncate(ann["data"])))
 
 
-@annotate(title="Fraction of Transport Reactions without GPR", type="percent")
```

Comment:
Is there any specific reason why you've deleted this function and re-added it?

To get a cleaner diff and proportionate line change statistics, in the future, please just append new code instead of putting it in the centre of a module.